internal/sdkv2provider/resource_cloudflare_email_routing_rule.go

```diff
@@ -46,8 +46,10 @@ func buildMatchersAndActions(d *schema.ResourceData) (matchers []cloudflare.Emai
 			action := item.(map[string]interface{})
 			ruleAction := cloudflare.EmailRoutingRuleAction{}
 			ruleAction.Type = action["type"].(string)
-			for _, value := range action["value"].([]interface{}) {
-				ruleAction.Value = append(ruleAction.Value, value.(string))
+			if val, ok := action["value"]; ok {
```

Comment:
what do we get from this additional check? given that `value` is required, how is this value never not set?

Reply:
It doesn't need right now, because `actions.value` is required value (in official docs).
But when use that API with `type: drop`, it's ok without `value`. (and it seem natural that there is no `value` when `type` is `drop`)

Of course, it is still `required` in API docs, and `value` can't even not set in terraform code.
Would it be better to remove this part?

Reply:
i'm happy either way; i was just more curious what this change, that is already working, was doing.